src/Object/Post.php

```diff
@@ -284,7 +284,7 @@ public function getTemplateData(array $conv_responses, string $formSecurityToken
 				'label' => DI::l10n()->t('Report post'),
 				'href'  => 'moderation/report/create?' . http_build_query(['cid' => $item['author-id'], 'uri-ids' => [$item['uri-id']]]),
 			];
-			if (!Network::isLocalLink($item['plink'])) {
+			if (!empty($item['uri']) && !Network::isLocalLink($item['uri'])) {
```

Comment:
The URI does not always contain an HTTP address. I guess best would be to include the `baseurl` field from the contact in the view for the posts and to check that. We could then even use that field for the label.

Reply:
Which contact?

Reply:
I suggest adding the `baseurl` here: https://github.com/friendica/friendica/blob/develop/static/dbview.config.php#L187

This can then also be used for the menu label.

Reply:
On both my nodes the `contact.baseurl` isn't consistently filled, same for `contact.gsid`. This also means it can't reliably be used for the Ignore Server menu label either.

Reply:
We can only block a server when we know the server. Also on my system only 4544 out of 3028402 contacts (ActivityPub as a network and not failed) don't have this field filled. My expectation is that the relationship will be similar on most systems.